### PR TITLE
Searchform Item Placeholder Visibility based on Label - Updated

### PR DESCRIPTION
**Pull request by iamibrahimriaz**
## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Security fix
- [x] Improvement
- [ ] New Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Text changes
- [ ] Other... Please describe:

## Description
How to reproduce the issue or how to test the changes

**Before:** 
When Label is empty: https://prnt.sc/UfFWUeZVL-Kc 

**After:**
When Label is empty: https://prnt.sc/rNEXJThM00u3 

**Default Label, Placeholder:**
Can't empty label & placeholder together: https://prnt.sc/wm_3BTs-RB8K | https://prnt.sc/mPy4BzrRvulY

## Any linked issues
Fixes #

## Checklist

- [x] My code follows the [WordPress coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)

### CODE REVIEW

File: includes/asset-loader/localized_data.php [hunk 1]
```diff
@@ -112,6 +112,8 @@ public static function get_listings_data() {
 			'lazy_load_taxonomy_fields'   => get_directorist_option( 'lazy_load_taxonomy_fields', false, true ),
 			'current_page_id'             => get_the_ID(),
 			'icon_markup'                 => '<i class="directorist-icon-mask ##CLASS##" aria-hidden="true" style="--directorist-icon: url(##URL##)"></i>',
+			'default_label'         	  => __( 'Label', 'directorist' ),
```
Comment (vairafiq) [inline includes/asset-loader/localized_data.php]: stuff needs to be specific please change it to `search_form_default_label` and `search_form_default_placeholder`

File: templates/search-form/fields/location.php [hunk 2]
```diff
@@ -46,7 +46,9 @@
 	?>
```
Comment (vairafiq) [inline templates/search-form/fields/location.php:46]: need to change the template version at the top `8.0.8.1`

**Review by vairafiq — APPROVED** [review]

File: templates/search-form/fields/location.php (hunk 2 above)
Comment (vairafiq) [reply templates/search-form/fields/location.php:46]: Need to change `@version`

**Review by vairafiq — CHANGES_REQUESTED** [review]

File: includes/asset-loader/localized_data.php (hunk 1 above)
Comment (iamibrahimriaz) [reply includes/asset-loader/localized_data.php]: Changed

File: templates/search-form/fields/location.php (hunk 2 above)
Comment (iamibrahimriaz) [reply templates/search-form/fields/location.php:46]: Changed

File: templates/search-form/fields/location.php [hunk 3]
```diff
@@ -2,7 +2,7 @@
 /**
  * @author  wpWax
  * @since   6.6
- * @version 8.0
+ * @version 8.0.8.1
```
Comment (obiPlabon) [inline templates/search-form/fields/location.php]: Version 8.0.9

**Review by obiPlabon — CHANGES_REQUESTED** [review]

**Review by vairafiq — APPROVED** [review]

**Review by obiPlabon — APPROVED** [review]

**Pull request merged**